### PR TITLE
Code style fixes in search.cpp

### PR DESCRIPTION
**Pull request by mcostalba**
Some code style triviality.

No functional change.

### CODE REVIEW

**Comment by snicolet** [comment]
Looks good to me, approved.

File: src/search.cpp [hunk 1]
```diff
@@ -302,21 +299,21 @@ void Thread::search() {
...
+                                : -make_score(ct, ct / 2));
 
   // Iterative deepening loop until requested to stop or the target depth is reached
   while (   (rootDepth += ONE_PLY) < DEPTH_MAX
          && !Threads.stop
          && !(Limits.depth && mainThread && rootDepth / ONE_PLY > Limits.depth))
   {
-      // Distribute search depths across the threads
-      if (idx)
+      // Distribute search depths across the slave threads
```
[the remark this answers is not in the document]
Comment (mcostalba) [reply src/search.cpp]: @joergoster yes, good point. I will change it.

**Comment by mcostalba** [comment]
Updated.

**Comment by snicolet** [comment]
Merged via https://github.com/official-stockfish/Stockfish/commit/4c57cf0ead29536504ca452b876d350a8e2edbdc.

**Pull request closed without merging**